src/main/java/org/vaadin/example/AppShell.java

```diff
@@ -3,6 +3,10 @@
 import com.vaadin.flow.component.page.AppShellConfigurator;
 import com.vaadin.flow.server.PWA;
 
+/**
+ * Use the @PWA annotation make the application installable on phones, tablets
+ * and some desktop browsers.
+ */
 @PWA(name = "Project Base for Vaadin Flow with Spring", shortName = "Project Base")
```

Comment:
Could you also remove the mention of "Flow"?

Reply:
Done.